docs/application/tizen-studio/release-notes/2-3-release-notes.md

```diff
@@ -0,0 +1,84 @@
+# Tizen Studio 2.3 Release Notes
+
+
+## IDE and Tools
+
+
+### New Features
+
+-   Tizen Studio
+	-   [JAVA 9](https://www.oracle.com/java/java9.html) support is added in Tizen Studio. Users can now download and install Tizen Studio along with JAVA 9.
+-   Base IDE
+	-   Tizen Studio is updated from [Eclipse Mars](https://www.eclipse.org/mars/) to [Eclipse Oxygen](https://projects.eclipse.org/releases/oxygen), to help user with improved performance and bug fixes.
+	-   CDT (C/C++ Development Tooling) is updated to 9.3.2 and the Debugger is migrated from CDI to DSF enabling faster debugging and easier extension.
+-	Emulator Manager
+	-   The Emulator Manager is updated to support JAVA 9.
+	-   [Intel&reg; HAXM](../../setup/hardware-accelerated-execution-manager.md) is upgraded to 7.0.0 to support Tizen Emulators on Windows 10 and Mac.
```

Comment:
Hello @karthikthecool. Sorry for confusing. `../setup/hardware-accelerated-execution-manager.md` is correct. Please remove one `../`. 

Hi, @Reni-90 After karthikthecool updates this PR, would you please check the update and "Approve" when everything is fine?